src/StagesCards.js

```diff
@@ -1,14 +1,41 @@
-import React from 'react';
+import React, { Component } from 'react';
 import './styles/App.css';
 
 
-
-
-
-
-
-
-
-
+class StagesCards extends Component {
+
+  render() {
+    if(this.props.stage === '') {
+      return ( <div></div> )
+    } else return (
+      <div className='stage-info-card' >
+          <section className='stage-card-main' >
+            <h1 className='stage-card-name'>{this.props.stage.name}</h1>
+            <img className='stage-card-image' src={this.props.stage.stage_image} />
+          </section>
+          <section className='stage-cards-details'> 
+            <p className='stage-cards-universe-name'>{this.props.stage.universe.name}</p>
+            <a href={this.props.stage.smash_wiki} className='stage-card-info-link'>Learn More</a>
+          </section>
+          <section className='stage-game-availability'>
+            {
+            Object.keys((this.props.stage)).map((key) => {
+              if (typeof this.props.stage[key] === 'boolean') {
```

Comment:
Is there a scenario where `this.props.stage[key]` **wouldnt** be a boolean? Generally you never want to change the data type of a value in JavaScript. e.g. `this.props.stage[key]` should always be a boolean and only ever equal true or false. If it ever changes into a string or number or other data type, you're going to be far more likely to run into bugs down the road.